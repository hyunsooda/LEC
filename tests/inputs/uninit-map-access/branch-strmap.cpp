#include <iostream>
#include <map>

using namespace std;

int main() {
    map<string, float> thisismymap;
    thisismymap["123"] = 999.99;
    float v = thisismymap["123"];

    int a = 123;
    if (a == 123) {
        printf("%d\n", thisismymap["888"]);
        a *= 2;
    } else {
        a *= 3;
    }
}
