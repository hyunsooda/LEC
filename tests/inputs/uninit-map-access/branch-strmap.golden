B[31mFound unitialized key access: ["branch-strmap.cpp":13:24]B[37m
    variable name: "thisismymap", allocated at: 7
